Sources/FluentUI_iOS/Components/Tab Bar/TabBarItemView.swift

```diff
@@ -307,10 +314,12 @@ class TabBarItemView: UIControl, TokenizedControl {
             let tintColor = tokenSet[.selectedColor].uiColor
             titleLabel.textColor = shouldTint ? tintColor : tokenSet[.unselectedTextColor].uiColor
             imageView.tintColor = shouldTint ? tintColor : tokenSet[.unselectedImageColor].uiColor
+            imageView.alpha = 1.0
         } else {
             let disabledColor = tokenSet[.disabledColor].uiColor
             titleLabel.textColor = disabledColor
             imageView.tintColor = disabledColor
+            imageView.alpha = style == .glass ? 0.6 : 1.0
```

Comment:
No urgency, since this already merged (and you gave me plenty of time to review it 😁), but I think it'd be nice to move the 0.6 constant into the constants extension of `TabBarItemTokenSet`. Just a nice-to-have next time one of us is touching this class!